Build/Common.Build.targets

```diff
@@ -145,6 +145,10 @@
     <Copy SourceFiles="@(DebugSymbolsProjectOutputGroupOutput->'%(FinalOutputPath)')"
           DestinationFolder="$(CopyOutputsToPath)Symbols" />
 
+    <Copy SourceFiles="@(SatelliteDllsProjectOutputGroupOutput)"
+          DestinationFiles="@(SatelliteDllsProjectOutputGroupOutput->'$(CopyOutputsToPath)UnsignedBinaries\%(TargetPath)')"
```

Comment:
Is TargetPath the right var here? From the docs: "The absolute path name of the primary output file for the build".